Makefile.include

```diff
@@ -658,7 +658,7 @@ term: $(TERMDEPS)
 	$(TERMPROG) $(TERMFLAGS)
 
 # Term without the pyterm added logging
-# TERMFLAGS must be exported for `jlink.sh term_rtt`.
+# TERMFLAGS must be exported for `jlink.sh term-rtt`.
```

Comment:
This looks weird: TERMFLAGS is not exported by target below. Might be a leftover from a previous refactoring ?